src/main/java/miu/cs425/controllers/VehicleController.java

```diff
@@ -1,10 +1,21 @@
 package miu.cs425.controllers;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import miu.cs425.models.Car;
+import miu.cs425.services.ICarService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/api/v1/vehicles")
 public class VehicleController {
 
+    @Autowired
+    private ICarService carService;
+    @PostMapping("/add")
```

Comment:
It should be followed the Restful API standard. You don't need to use path `/add`. (Restful API is just based on kind of request method)

Reply:
Please add API document like this: https://github.com/crisnguyenvn/crs-service/blob/dc7a0a4613a49217c446bca7cbacc2aaba5827df/src/main/java/miu/cs425/controllers/UserController.java#L23

Access to URL to view API document: http://locahost:8080/swagger-ui/index.html

![image](https://github.com/user-attachments/assets/78c20676-6b69-4ead-ab88-0e6d2fec5371)

Reply:
I've standardized Post request path and added it to the API document, please review it again!